yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -243,8 +243,12 @@ func (yb *TargetYugabyteDB) InitConnPool() error {
 		SessionInitScript: getYBSessionInitScript(yb.tconf),
 	}
 	yb.connPool = NewConnectionPool(params)
-	redactedParams := params
-	redactedParams.ConnUriList = utils.GetRedactedURLs(redactedParams.ConnUriList)
+	redactedParams := ConnectionParams{
```

Comment:
We're logging the full struct below, so this will mean we'll have to come back to keep updating this copy here whenever we add a new field. 

Let's go with `redactedParams := *params` ?